packages/preview/isc-hei-report/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "isc-hei-report"
+version = "0.1.0"
+compiler = "0.11.0"
+entrypoint = "isc_template_report.typ"
+repository = "https://github.com/ISC-HEI/ISC-report"
+authors = ["Pierre-andré Mudry"]
+license = "MIT"
+description = "An official report template for the 'Informatique et systèmes de communication' (ISC) bachelor degree programme at the School of Engineering (HEI) in Sion, Switzerland."
+keywords = ["report", "ISC", "university", "Informatique et systèmes de communication", "HES-SO Valais", "Sion", "HEI", "School of engineering"]
+categories = ["report"]
+disciplines = ["computer-science", "engineering"]
+
+[template]
+path = "template"
+entrypoint = "report.typ"
+thumbnail = "thumbnail.png"
```

Comment:
Git did something weird apparently, because the previous version appears in this PR too even if it is already merged. Can you please have a look at what happened and try to fix it, to keep the history cleaner?

Reply:
I tried to figure out a way to do it but I don't even know where to look. If I can't do I, will simply start from a fresh version.